src/main/resources/application-dev.properties

```diff
@@ -0,0 +1,42 @@
+#jpa:
```

Comment:
Preferirei che questo file fosse in una direcotry config nella radice del progetto, in modo da non finire nel jar.

Reply:
Ok, dobbiamo farlo per tutti i profili? Il file in questione riguarda il profilo dev e il percorso attuale è quello utilizzato di default da Spring

Reply:
Se ben ricordo spring boot è in grado di cercare i file di property anche nella directory /config in base alla current working direcotry.
Dentro il jar vorrei solo le property di default. Tutte le property che variano in base agli ambienti vengono iniettate da variabili d'ambiente.

Reply:
risolto